packages/caravan-psbt/README.md

```diff
@@ -30,8 +36,6 @@ A set of utilities for working with PSBTs.
       - [`public removePartialSig`](#public-removepartialsig)
       - [`static PsbtV2.FromV0`](#static-psbtv2fromv0)
     - [`function getPsbtVersionNumber`](#function-getpsbtversionnumber)
-    - [`abstract class PsbtV2Maps`](#abstract-class-psbtv2maps)
-      - [`public copy`](#public-copy)
```

Comment:
This wasn't actually being exported by the module, so it has been removed from the documented API.